scripts/feature_toggle_report_generator.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+
+import datetime
+import io
+import os
+import re
+
+import click
+import jinja2
+import yaml
+
+
+class Ida(object):
+
+    def __init__(self, name):
+        self.name = name
+        self.toggle_states = {}
+        self.annotation_report_path = None
+
+    def add_toggle_data(self, dump_file_path):
+        """
+        Given the path to a file containing the SQL dump for a
+        feature toggle type in an Ida, parse out the information relevant
```

Comment:
I can see why you'd want to capitalize it this way for the class name to match PEP 8 recommendations, but it really should be IDA (Independently Deployed Application).

Reply:
Yeah, having an acronym in the python tokens has been driving me crazy. Will do.